src/services/groups.service.ts

```diff
@@ -0,0 +1,96 @@
+import { PrismaClient, Group } from '@prisma/client';
```

Comment:
I think the name of this file should be group.service insted of groups.services for consistency